e2e_tests/helpers.py

```diff
@@ -48,6 +72,9 @@ def run_cli_command(
     if yes:
         command = f"yes Y | {command}"
 
+    # Prefix the command with 'poetry run' to ensure it runs in the Poetry shell
+    command = f"poetry run {command}"
```

Comment:
This makes running E2Es locally less error prone by forcing them to run in the poetry shell that's guaranteed to point to the correct `darwin-py` installation

Reply:
btw, python also has [sys.executable](https://docs.python.org/3/library/sys.html#sys.executable) 😄

Reply:
I didn't know this, it's much nicer. thank you!